### PR TITLE
Amended and addressed feedback for the Encryption feature documentation.

### PR DESCRIPTION
**Pull request by alexemc**
# Description
Amended and addressed feedback for the Encryption feature documentation.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [x] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

**Review by rsedlock1958 — DISMISSED** [review]
I have reviewed this. No edits are required.

**Review by rsedlock1958 — DISMISSED** [review]
I have reviewed this. No edits are required.

File: content/docs/secure/encryption/_index.md [hunk 1]
```diff
@@ -118,8 +120,11 @@ Refer to [Vault Configuration section](vault) for minimal configuration steps re
 ## Kubernetes Worker Hosts Requirements
 
 - Each Kubernetes worker host should have SSH server running.
-- The server should have SSH public key authentication enabled for user *root*. 
-- The server should remain running all the time whenever an application with an encrypted volume is running on the host.
+- SSH server should have SSH public key authentication enabled for user *root*. 
+- SSH server should remain running all the time whenever an application with an encrypted volume is running on the host.
 > **NOTE:** Stopping the SSH server on the worker host makes any encrypted volume attached to this host [inaccessible](troubleshooting#ssh-stopped).
+- Each Kubernetes worker host should have commands `fusermount` and `mound.fuse`. They are pre-installed in most Linux distros.
```
Comment (anathoodell) [inline content/docs/secure/encryption/_index.md:126]: and 'mound.fuse' or 'mount.fuse'?

**Review by rsedlock1958 — COMMENTED** [review]
I have reviewed this. No edits are required.

**Review by anathoodell — APPROVED** [review]

**Comment by alexemc** [comment]
Can you approve it as well?
> I have reviewed this. No edits are required.

**Comment by rsedlock1958** [comment]
I cannot access it now. You will have to request my review.

**Review by rsedlock1958 — APPROVED** [review]

**Pull request merged**